polynote-kernel/src/main/scala/polynote/kernel/lang/python/PythonInterpreter.scala

```diff
@@ -524,7 +532,34 @@ class PythonInterpreter(val kernelContext: KernelContext, dependencyProvider: De
     cellContext: CellContext,
```

Comment:
Ugh, github won't let me comment on its line but looks like we can remove `// TODO: can parameter hints be implemented for python?` :)